modules/styleguide/assets/js/styleguide-app-initiator.js

```diff
@@ -29,9 +34,20 @@
 	 * @return {Object|null}
 	 */
 	function getStyleguideWidget() {
-		styleguideWidget = document.querySelector( '.dialog-styleguide-message' );
+		return document.querySelector( '.dialog-styleguide-message' );
+	}
+
+	/**
+	 * Create root if the container element is available.
+	 */
+	function maybeCreateRoot() {
+		const widget = getStyleguideWidget();
+
+		if ( styleguideRoot || ! widget ) {
+			return;
+		}
 
-		return styleguideWidget;
+		styleguideRoot = createRoot( widget );
 	}
```

Comment:
Is this function called only once?
maybe move extract from the function or rename?